apps/vs-code-designer/src/app/utils/binaries.ts

```diff
@@ -68,17 +67,20 @@ export async function downloadAndExtractDependency(
     fs.mkdirSync(tempFolderPath, { recursive: true });
     fs.chmodSync(tempFolderPath, 0o777);
 
-    // Download the compressed dependency
-    await new Promise<void>((resolve, reject) => {
-      executeCommand(ext.outputChannel, undefined, 'echo', `Downloading dependency from: ${downloadUrl}`);
-      // TODO: @mireed: change the download to be async and not block.
-      // TODO: @mireed: the NPM Request library is deprecated, change to use a different library.
-      // Get the Dependency Timout setting value, which is in seconds and convert to msec for the request library.
-      const dependencyTimeoutMsec = getDependencyTimeout() * 1000;
-      const downloadStream = request({ url: downloadUrl, timeout: dependencyTimeoutMsec }).pipe(fs.createWriteStream(dependencyFilePath));
-      downloadStream.on('finish', async () => {
-        await executeCommand(ext.outputChannel, undefined, 'echo', `Successfullly downloaded ${dependencyName} dependency.`);
+    const downloadPromise = axios({
+      method: 'get',
```

Comment:
NIT: We have a constant for this `HTTP_METHODS.GET`

Reply:
awesome... fixed